user-interface/src/lib/components/uswds/modal/SubmitCancelButtonGroup.test.tsx

```diff
@@ -100,7 +100,6 @@ describe('Submit cancel button group tests', () => {
     expect(cancel.className).toContain(UswdsButtonStyle.Unstyled);
     expect(cancel.className).toContain('test-class-two');
     expect(cancel.className).not.toContain('padding-105');
-    expect(cancel.className).not.toContain('text-center');
   });
```

Comment:
_Just giving context_:  We altered the component so that the text is always centered.  We couldn't think of a situation where we wouldn't want the button text centered.